Tools/px_generate_pubsub_graph.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+import os
+import glob
+import re
+import codecs
+
+from graphviz import Digraph
+from px4params import scope, cmakeparser
```

Comment:
cmakeparser is getting nuked here (https://github.com/PX4/Firmware/pull/6385) , it didn't know how to parse include statements, so it is better to just use cmake to parse the cmake  files. 

To do this, make this script take a list of modules like I did in the PR. Then you can tie this script into cmake to do the scoping and pub sub graph creation.